config/locales/en.yml

```diff
@@ -5051,6 +5064,7 @@ en:
     others are just down the street! %{site_name} Places let you browse life
     from around the planet.  Start exploring!
   then_keybord_sequence: then
+  there_are_no_current_ids: There are no current IDs of any narrowed parents.
```

Comment:
Again, I don't know what a narrowed parent is, so you're going to have to explain it in a comment.

Reply:
added comments

Reply:
I still don't understand what this means. If you move genus Homo to family Canidae, then is family Homonidae "narrowed" because it has fewer descendants? Is there no better word for this, or a word that we already use? I feel like you're adding several new pieces of jargon that make our complicated taxonomic situation even harder to understand, and that's just in English.

Reply:
yes - if you were to move Homo to Canidae you would be narrowing Homonidae. Existing IDs of Homonidae that were made thinking that the branch includes Homo would be reinterpreted by iNat as disagreements with IDs of Homo. These are clashes. If there are >10 clashes you should split the narrowed taxa (e.g. Homonidae). If you want to propose with different semantics or policy happy to consider

Reply:
I don't think you need new meanings (i.e. semantics), I think you need more descriptive terms, but I don't have better suggestions. Personally I prefer to stick to the tree metaphor (e.g. prune, graft), so I might go with "thin" instead of "narrow" as in thinning the branches. I wish there were more tree-like terms for ancestor / descendant and parent / child.

I do think this is so complicated that you should probably link to your documentation in these comments.